src/main/java/ua/price/bdd_solution/bdd_pages/pages/SearchPageResults.java

```diff
@@ -0,0 +1,113 @@
+package ua.price.bdd_solution.bdd_pages.pages;
+
+import net.serenitybdd.core.annotations.findby.FindBy;
+import net.serenitybdd.core.pages.WebElementFacade;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import ua.price.bdd_solution.bdd_pages.base_page.BasePage;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class SearchPageResults extends BasePage {
+    @FindBy(xpath = ".//div[contains(@class, 'product-block type')]")
+    private List<WebElementFacade> searchResultsList;
+
+    @FindBy(xpath = ".//a[contains(@class,'btn-orange')]")
+    private WebElementFacade itemPageButton;
+
+    @FindBy(xpath = ".//a[@data-producer-alias='samsung']")
```

Comment:
Do we always look for a WebElement related to Samsung? Can we change it to avoid hard-coding here?

Reply:
Done